src/main/java/org/openstreetmap/atlas/geography/atlas/items/AtlasEntity.java

```diff
@@ -194,6 +199,66 @@ public String toDiffViewFriendlyString()
      */
     public abstract LocationIterableProperties toGeoJsonBuildingBlock();
 
+    public abstract JsonObject asGeoJsonFeature();
+
+    /**
+     * A method that creates properties for a GeoJSON Feature from the tags.
+     *
+     * @return A GeoJSON properties object that is to be put in a Feature.
+     */
+    protected JsonObject geoJsonProperties()
+    {
+        final JsonObject properties = new JsonObject();
+        getTags().forEach(properties::addProperty);
+        properties.addProperty(GeoJsonUtils.IDENTIFIER, getIdentifier());
+        properties.addProperty(GeoJsonUtils.OSM_IDENTIFIER, getOsmIdentifier());
+        properties.addProperty(GeoJsonUtils.ITEM_TYPE, String.valueOf(getType()));
+
+        final Optional<String> shardName = getAtlas().metaData().getShardName();
+        shardName.ifPresent(shard -> properties.addProperty("shard", shard));
+
+        final Set<Relation> relations = relations();
+        if (relations.size() > 0)
+        {
+            final JsonArray relationsArray = new JsonArray();
+            properties.add("relations", relationsArray);
+            for (final Relation relation : relations)
+            {
+                final JsonObject relationObject = relation.geoJsonProperties();
+                relationsArray.add(relationObject);
+                final RelationMemberList members = relation.members();
+                final JsonArray membersArray = new JsonArray();
+                relationObject.add("members", membersArray);
+                for (final RelationMember member : members)
+                {
+                    final JsonObject memberObject = new JsonObject();
+                    membersArray.add(memberObject);
+                    final AtlasEntity entity = member.getEntity();
+                    if (entity != null)
+                    {
+                        final long identifier = entity.getIdentifier();
+                        memberObject.addProperty(GeoJsonUtils.IDENTIFIER, identifier);
+                    }
+                    else
+                    {
+                        // We shouldn't get here, but if we do, let's know about it in the data...
+                        memberObject.addProperty(GeoJsonUtils.IDENTIFIER, "MISSING");
+                    }
+
+                    // Sometimes a member doesnt have a role. That's normal.
+                    final String role = member.getRole();
+                    if (role != null)
+                    {
+                        // And sometimes the role is "", but we should keep it that way...
+                        memberObject.addProperty("role", role);
```

Comment:
Do we want to add the type (Edge, Node, etc.) of member as well or does that not matter here?

Reply:
++1

Reply:
done 👍